src/demo.ts

```diff
@@ -134,3 +139,4 @@ win.show();
 win.setWindowState(WindowState.WindowActive);
 
 (global as any).win = win; // To prevent win from being garbage collected.
+(global as any).systemTray = tray; // To prevent system tray from being garbage collected.
```

Comment:
`tray` is garbage collected since `QSystemTrayIcon` isn't a part of `window`, `. Is there a better way to prevent grabage collection?

Reply:
I don't think there is, plus it's cool, because if you're creating a renderer (like React or Angular) it'd make more sense that it SHOULD be garbage collected when in a test environment (such as when using https://github.com/ng-qt/ng-test combined with https://jestjs.io, even though Jest can force exit a process that haven't finished)